nilearn/decomposition/_multi_pca.py

```diff
@@ -111,28 +109,33 @@ class _MultiPCA(_BaseDecomposition):
 
     Attributes
     ----------
-    masker_ : instance of MultiNiftiMasker
+    masker_ : instance of MultiNiftiMasker or \
```

Comment:
doc strings for attributes seem to be VERY similar between classes: maybe let's try to refactor them using _utils/docs.py

Reply:
Indeed I was just unsure how best to parameterize this because this `masker_` docstring would be different from that in other places like in maskers or decoders.